data/boomr_2020.csv

```diff
@@ -0,0 +1,409 @@
+komnr;komnr_navn;bydel;bydel_navn;bohf;bohf_navn;boshhn;boshhn_navn;borhf;borhf_navn;kommentar
```

Comment:
Kan denne hete "boomr", så vi slipper å gi nytt navn hvert år?

Reply:
I agree that it is much easier to have a static file name.  However, it was an intentional decision to have the year attached to the file name so that we know the content is updated as of january of that year, same as the ICD-10 file.  Alternatively if we insert a comment on the first line (so that it is obvious and easy to find) of the files to say "updated as of 1.1.2021" for example.  Would that be a good solution? @Tovjoh

Reply:
Jeg tror ikke csv-filer kan ha kommentarer, så kanskje like greit å beholde det slik dere har tenkt? Ulempen vil vel være at alle makroer og annen kode som bruker denne csv-filen også må oppdateres hver gang filen oppdateres.

Reply:
Har endret navn på filen til 'boomr', også er det lagt inn en kommentar på rad 2 med når den ble sist oppdatert. 
Når vi leser inn csv-filen til SAS kan vi si at den skal starte med rad 3. Kommentaren kommer altså ikke med inn i SAS, men er lett å sjekke i csv-filen.